src/app.rs

```diff
@@ -414,6 +518,33 @@ impl eframe::App for RustyAutoClickerApp {
             }
         }
 
+        // move to target position
+        if self.is_moving {
+            #[cfg(debug_assertions)]
+            println!(
+                "Moving from {:?}/{:?} towards: {:?}/{:?}",
+                mouse.coords.0.to_f64(),
+                mouse.coords.1.to_f64(),
+                click_x,
+                click_y
+            );
+            move_to(
+                self.app_mode,
+                self.click_position,
+                (click_x, click_y),
+                self.is_moving_humanlike,
+                (mouse.coords.0.to_f64(), mouse.coords.1.to_f64()),
+                movement_delay_in_ms,
+                self.rng_thread.clone(),
+            );
+            if mouse.coords.0.to_f64() == click_x && mouse.coords.1.to_f64() == click_y {
```

Comment:
Can this be moved inside of `move_to()` or is rustlang's borrow checker not allowing it?